corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -142,21 +141,21 @@
             <div class="col-sm-offset-4 col-md-offset-3 col-lg-offset-2 col-sm-8 col-md-9 col-lg-10 controls">
                 <button type="submit" class="btn btn-lg btn-primary" data-bind="
                     click: save,
-                    disable: (saveState() === Exports.Constants.SAVE_STATES.SAVING ||
-                              saveState() === Exports.Constants.SAVE_STATES.SUCCESS)
+                    disable: (saveState() === hqImport('export/js/const.js').SAVE_STATES.SAVING ||
+                              saveState() === hqImport('export/js/const.js').SAVE_STATES.SUCCESS)
```

Comment:
`hqImport` as a function just looks up a memoized value; behind the scenes `hqDefine` adds a value to a global object and `hqImport` looks it up from the global object (while adding some error checking and other lightweight niceties). I wouldn't worry about calling `hqImport` being slow.